plugins/webrtc/src/main.ts

```diff
@@ -727,15 +715,17 @@ export async function fork() {
     }
 }
 
-class WebRTCBridge extends ScryptedDeviceBase implements BufferConverter {
+class WebRTCBridge extends ScryptedDeviceBase implements MediaConverter {
```

Comment:
it looks like this was consolidated into webrtc bridge, which is itself just a buffer converter. the plugin is a buffer converter as well. let's consolidate all 3.

Reply:
I actually added media converter for webrtc plugin and rebroadcast plugin to better handle multiple conversions. just had to wait a reasonable amount of time for servers to update.